kalite/contentload/management/commands/annotate_content_items.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
Management command - little more than a convenience wrapper around the annotate_content_models method.